### PR TITLE
Update 12-Hash.md with hashmap image

### LINKED ISSUE #28: Add image of hashmap in 12-Hash.md
In this file, we could add an image demonstrating Hashmap. It could potentially help students to understand it.

### PR DESCRIPTION
**Pull request by Milo-Zhao**
Fixes #28 , the image is original work.

### CODE REVIEW

**Comment by Milo-Zhao** [comment]
Fix issues #28

**Review by onzfonz — CHANGES_REQUESTED** [review]
Also write fixes #XX in the description of the pull request, not as part of the title

File: docs/labs/12-Hash.md [hunk 1]
```diff
@@ -41,7 +41,9 @@ Hashtables allowed us to search and store items into a collection in constant O(
 which means that no matter the size of the hashtable,
 you would be able to count on the running time not increasing
 (as long as the number of buckets is roughly equal to the
-number of items and you have a full hash function yada-yada)
+number of items and you have a full hash function yada-yada).
+This is a picture that can help you understand hashmap more intuitively:
```
Comment (onzfonz) [inline docs/labs/12-Hash.md:45]:
```suggestion
Here's a picture that may help you understand a `HashMap` more intuitively:
```

**Review by onzfonz — CHANGES_REQUESTED** [review]

File: docs/labs/12-Hash.md (hunk 1 above)
Comment (Milo-Zhao) [reply docs/labs/12-Hash.md:45]: I can do that!

**Comment by onzfonz** [comment]
Edit the place that says this image is original work at the top of the pull request, just below the title

**Review by onzfonz — APPROVED** [review]

**Pull request merged**